### ISSUE TITLE
Keycloak closes connection after 60s

### ISSUE THREAD

**Issue by gt-novelt** [marcospereirampj/python-keycloak#36]
By default Keycloak closes a connection unused for 60s (source: [tomcat's documentation](https://tomcat.apache.org/tomcat-9.0-doc/config/http.html)).

This is not an issue for the current release of python-keycloak since `KeycloakAdmin` and `KeycloakOpendID` are meant to be initialized each time you want to call Keycloak. By instantiating a new version of `KeycloakAdmin`/`KeycloakOpendID`, you are also initiating a new `ConnectionManager` thus a new `requests.Session` thus a new connection to Keycloak server.
To put it simply, _python-keycloak_ does not take advantage of `requests.Session`'s persistent connection.

That being said, If PR #32 is accepted, then only one instance of `KeycloakAdmin`/`KeycloakOpendID` is needed. The persistent connection is being used. But since Keycloak closes the connection after 60s of inactivity, we get a 
> 'Connection aborted.', RemoteDisconnected('Remote end closed connection without response'
